docs/versioned_docs/version-2.11.0/additional-info/kb.md

```diff
@@ -1,6 +1,11 @@
 ---
 id: kb
 title: Knowledge Base
+keywords: 
+ - Knowledge base
+ - OpenEBS Knowledge base
```

Comment:
Let's see if we can add a few more keywords.

Reply:
Added few more.